root/release_group/ReleaseGroupIndex.js

```diff
@@ -131,8 +131,11 @@ const ReleaseGroupIndex = ({
               <thead>
                 <tr>
                   {$c.user_exists
-                    ? <th className="checkbox-cell"><input type="checkbox" /></th>
-                    : null}
+                    ? (
```

Comment:
The `? (` here can go on the previous line and the innards de-intended to match the `) :` below. Or `: null}` can be moved to its own line to match the `? <th`.